menu_scene.py

```diff
@@ -0,0 +1,41 @@
+from enum import Enum
+
+from base_scene import BaseScene
+from event_system import EventSystem
+
+class MenuScene(BaseScene):
+    class Events(Enum):
+        CREATE_CHARACTER = 1
+        SELECT_CHARACTER = 2
+        LIST_CHARACTER = 3
+        SELECT_ENEMY_CHARACTER = 4
+        REMOVE_CHARACTER = 5
+
+    def render(self):
+        print("""
+Main Menu:
+
+1) create character
+2) select character
+3) list characters
+4) select enemy (optional)
+5) remove character
+
+        """)
+
+    def loop(self):
+        option = input('Select option: ')
+        event = None
+        if(option == '1'):
```

Comment:
can this `"1"` be a number instead of a string

Reply:
indeed, but for now I think a str is enough since `input` returns a str, and we if do a number casting like `int(input())` that can trigger an exception if the user types a letter

Reply:
ok